spec/quby/answers/entities/table_backend/table_dimension_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+describe Quby::Answers::Entities::TableBackend::TableDimension do
+  describe 'described_class::AcceptsAnythingRange' do
```

Comment:
Klinkt iets minder als een range (geen begin/eind), misschien is het meer de https://en.wikipedia.org/wiki/Universal_set?